src/com/comeet/RoomsDao.java

```diff
@@ -166,6 +167,9 @@ public RoomsDao(ExchangeService service) {
         } catch (ClassNotFoundException e) {
             // TODO: Auto-generated catch block.
             e.printStackTrace();
+        } catch (Exception e) {

```

Comment:
Please don't swallow Exception. Wrap or re-throw so that errors are reported on the http endpoint.